mgmtsystem/readme/CREDITS.rst

```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
+
+* Camptocamp
```

Comment:
I'm quite sure Camptocamp was not involved in the original development of this module.
Can you elaborate on this please?

Reply:
Hi @dreispt 

It has.

@dzungtran89 is working for Trobz that has been commissioned by Camptocamp to work on this migration.

Reply:
Hi @dreispt indeed; however it sponsored its migration; of for you if `s/development/migration/` ?

Reply:
@nilshamerlinck Yes, it is OK to do that. Please be specific and say that "The Odoo 14 migration of this module was...".
Sorry for nitpicking on this, but the current wording gives the wrong message and is unfair to the original developers and other migration funders.
Thanks.